bittensor/core/subtensor.py

```diff
@@ -223,14 +225,15 @@ def _get_substrate(self):
             except (AttributeError, TypeError, socket.error, OSError) as e:
                 logging.warning(f"Error setting timeout: {e}")
 
-        except ConnectionRefusedError:
+        except ConnectionRefusedError as error:
             logging.error(
                 f"Could not connect to {self.network} network with {self.chain_endpoint} chain endpoint.",
             )
             logging.info(
                 "You can check if you have connectivity by running this command: nc -vz localhost "
-                f"{self.chain_endpoint.split(':')[2]}"
+                f"{self.chain_endpoint}"
             )
+            raise ConnectionRefusedError(error.args)
```

Comment:
```suggestion
            raise
```

Then you don't have to worry about the `error` var.

Reply:
I want to show the user not only the stack, but also the code with the message